src/ButtonComponent/types.ts

```diff
@@ -0,0 +1,5 @@
+export interface ButtonProps {
+  content?: string;
+  type?: string;
+  // type: 'primary' | 'secondary' | 'natural';
```

Comment:
By `type` you mean variant?, if so then using variant would be better as it's far more common, also as you commented using string laterals or enums is also better.

Reply:
no by type I mean -- (primary, secondary, etc..)
I will use variants to the switch between (filled, Ghost, text)

Reply:
then `color` would be more convenient, I usually refer to the material ui site and see how they named their props.

Reply:
thanks for alerting me to that, I will consider it in the next commits